### PR TITLE
Mobile styles

### PR DESCRIPTION
**Pull request by cronin4392**
My main goal here was to get the homepage on smaller devices to look and function well enough. The homepage in it's current state on mobile does not scale correctly and has readability issues. In this PR I diverged from how CSS was being written in the repo and used the [BEM](https://css-tricks.com/bem-101/) naming syntax. I did this for a couple reasons:

- It makes my changes less likely to break other styles. By using only class names I created I am sure that no other pages are affected.

- BEM naming convention is a well trusted and tested way of writing CSS. If the authors of the main repository are inline with this method I think future PRs could be authored to get the CSS more inline with BEM which would make it more maintainable.

### CODE REVIEW

**Comment by b-g** [comment]
Hi @cronin4392, 

Many thanks for this! I just checked it ... the landing page looks definitely a lot nicer now! However I think we could be a bit more radical :)

**Landing**
* sticker header nav bar should be of smaller height, basically a single line of text  e.g. OpenMoji, Library, Participate and then a hamburger menu for the rest

* Section with animate emojis on scroll ... still not a beauty :) Maybe simply remove or adapt the size of the emojis better to mobile screen sizes?

**General Qs**
* Would you also be willing to tweak the other pages? // high prio

* Refactor the css to BEM naming syntax // nice to have
* Refactor the html markup to individual components e.g. react ... with the crazy redundancy in every html file it is quite a mess  // nice to have

Long story short a "full" iteration contribution would be way better for us in terms of project management and team spirit with the others. Happy to discuss this also in a call if you want, or in this issue. Let me know.

🙏 !

**Comment by b-g** [comment]
Hi @cronin4392. Ping! Hope I did not scary you off :) Best, Benedikt

**Comment by cronin4392** [comment]
Hey @b-g! So sorry. I've been busy with work things. I can respond to each of your items when I have more time but in general my thought was this PR would be to get the homepage in a decent state for mobile and then follow up with more radical PRs.

I briefly looked at the other pages on the site and they have a pretty different design. Would be interested in hearing if that was intentional or if homepage was in that same look and then redesigned?

**Comment by b-g** [comment]
Hi @cronin4392, no worries! We are planning to have a quick sprint to improve the current situation over the 🎄 break. See #9 for the discussion. If you have afterwards some time and energy (or if you want to join) ... you are very welcome to contribute.
